yt/utilities/answer_testing/framework.py

```diff
@@ -368,6 +370,18 @@ def sim_dir_load(sim_fn, path=None, sim_type="Enzo", find_outputs=False):
     )
 
 
+def lax_getter(key, dict_like):
```

Comment:
This should be removed before merging (used to handle mismatch in answer tests keys)

Reply:
So if this is going to get merged in the triage meeting today should it be deleted now?

Reply:
No, it is deleted in #3194 by commit 7771d30750474b404e7ca40ad25bd3d4f917e79d!